api/models/Detainees.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  autoPK: false,
+  attributes: {
+    person_id: {
+      type: 'string',
+      required: true,
+      unique: true,
+      primaryKey: true
+    },
+    cid_id: {
+      type: "integer"
+    },
+    gender: {
+      type: "string",
+      enum: ["male", "female"]
+    },
+    nationality: {
```

Comment:
we can totally ignore the nationality capture for now, we don't have any current user stories that talk about doing anything with it